grr/server/grr_response_server/gui/api_integration_tests/metadata_test.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Lint as: python3
+# -*- encoding: utf-8 -*-
```

Comment:
Since Python 3, UTF-8 is the default encoding, no need to specify that.

Reply:
Removed it. Thank you!